.github/workflows/gmc-helm.yaml

```diff
@@ -56,14 +56,30 @@ jobs:
       - name: Install GMC
         run: |
           echo "should_cleanup=true" >> $GITHUB_ENV
-          ## IMPORTANT NOTICE: need to replace gmc images with {{env.DOCKER_REGISTRY}} and {{env.VERSION}}
-          cd microservices-connector
-          if ! helm install --create-namespace --namespace {{ env.SYSTEM_NAMESPACE}} --wait \
+          USER_ID=$(whoami)
+          HFTOKEN=$(cat /home/$USER_ID/.cache/huggingface/token)
+          pushd microservices-connector/helm
```

Comment:
Can we add this info into microservices-connector/helm/README.md as well?

Reply:
done